OpenAFSLibrary/keywords/path.py

```diff
@@ -157,7 +175,7 @@ def symlink(self, src, dst, code_should_be=0):
             logger.info("os.symlink(): %s" % e)
             code = e.errno
         logger.info("os.symlink()=%d" % code)
-        if code != int(code_should_be):
+        if _cvt_errno_parm(code_should_be) != code:
```

Comment:
Curious? why did you flip the evaluation? It was "code != int(code_should_be)" and now it's _cvt_errno_parm(code_should_be) != code" ? I thought we'd just replace the "int" with the new convert function.

I take it _cvt_ means "convert"?

Reply:
thanks for fixing.